client/src/api/queries/my.ts

```diff
@@ -20,9 +20,11 @@ export const myEventQueryOptions = {
 };
 
 const getHostEvents = async (organizationId: number): Promise<Event[]> => {
-  return await fetcher.get<Event[]>(`organizations/${organizationId}/events/owned`);
+  return await fetcher.get<Event[] & EventType>(`organizations/${organizationId}/events/owned`);
```

Comment:
p2.
해당 타입은 불필요할 것 같아요!

```suggestion
  return await fetcher.get<Event[]>(`organizations/${organizationId}/events/owned`);
```

Reply:
넵 ! 

bfb111b31cb936944aeec758e555af6f607d1ffc